toolchain/openocd-mimiker/Makefile

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/make -f
+
+VER := 0.10.0+dev
+OPENOCD := openocd-$(VER)
+
+all: binary
+
+download:
+	git clone https://git.code.sf.net/p/openocd/code $(OPENOCD)
+
+configure:
+	rm -rf openocd-build
+	mkdir openocd-build
+	cd $(OPENOCD) && \
+	    ./bootstrap
+	cd openocd-build && \
+	    ../$(OPENOCD)/configure \
+	        --prefix=/usr \
+		--enable-jlink 
+	touch $@-stamp
+
+build: configure
+	cd openocd-build && make -j$(shell nproc --all) LDFLAGS="-Wl,--as-needed"
+	touch $@-stamp
+
+install: build
+	cd openocd-build && make install DESTDIR=$(PWD)/debian/tmp
+	cd $(PWD)/debian/tmp && \
+		mv usr/bin/openocd usr/bin/openocd-mimiker && \
+		mv usr/share/openocd/ usr/share/openocd-mimiker
```

Comment:
I don't think it's going to work. Most likely there're some strings embedded into openocd binary that refer to `/usr/share/openocd` directory.